elm/utilities/try_import.py

```diff
@@ -0,0 +1,32 @@
+"""ELM import utility."""
+import importlib
+import logging
+from warnings import warn
+
+logger = logging.getLogger(__name__)
+
+
+def try_import(package_name):
+    """Function to import packages needed for
+    postgres and aws.
+
+    Parameters
+    ----------
+    package_name : str
+        Name of package to import.
+
+    Returns
+    -------
+    p : package
```

Comment:
This function doesn't return anything

Reply:
Doesn't this return the import? `p = importlib.import_module(package_name)
        return p`